docs/backend-system/building-backends/01-index.md

```diff
@@ -10,3 +10,169 @@ description: Building backends using the new backend system
 > system, see [migrating](./08-migrating.md).
 
 # Overview
+
+A minimal Backstage backend is very lightweight. It is a single package with a `package.json` file and a `src/index.ts` file, not counting surrounding tooling and documentation. The packages is typically placed withing the `packages/backend` folder of a Backstage monorepo, but that is up to you. The backend package is part of any project created with `@backstage/create-app`, so you typically do not need to create it yourself.
+
+You'll get a brief description here, but for a more in-depth explanation of the role of backends in the backend system, see [the architecture section](../architecture/02-backends.md).
+
+When you create a new project with `@backstage/create-app`, you'll get a backend package with a `src/index.ts` that looks something like this:
+
+```ts
+import { createBackend } from '@backstage/backend-defaults';
+import { appPlugin } from '@backstage/plugin-app-backend';
+import { catalogPlugin } from '@backstage/plugin-catalog-backend';
+import { catalogModuleTemplateKind } from '@backstage/plugin-catalog-backend-module-template-kind';
```

Comment:
😛

Reply:
D: :O

Reply:
Reverted this so we can get this shipped :>